ext/sockets/sockets.c

```diff
@@ -1666,17 +1664,18 @@ PHP_FUNCTION(socket_sendto)
 	//struct sockaddr_ll      sll;
 #endif
 	int					retval;
-	size_t              buf_len, addr_len;
+	size_t              buf_len;
 	zend_long			len, flags, port = 0;
 	bool           port_is_null = 1;
-	char				*buf, *addr;
+	char				*buf;
+	zend_string                     *addr;
```

Comment:
I'm not sure the indent makes sense. Especially as the other ones are tab indented...
```suggestion
	char				*buf;
	zend_string *addr;
```